clients/include/testing_ormqr_unmqr.hpp

```diff
@@ -590,9 +590,8 @@ void testing_ormqr_unmqr(Arguments& argus)
 
     // validate results for rocsolver-test
     // using s * machine_precision as tolerance
-    int s = left ? m : n;
     if(argus.unit_check)
-        ROCSOLVER_TEST_CHECK(T, max_error, s);
+        ROCSOLVER_TEST_CHECK(T, max_error, (left ? m : n));
```

Comment:
What was this one complaining about? It seems like it was better before, where you had a name for `left ? m : n` that you use to refer to the value in the comments.

If it's just complaining that the variable scope could be reduced, maybe put it inside the `if(argus.unit_check)` block instead.

Reply:
This was one of the unusedVariable warnings. I guess it had trouble sensing that `s` is used in the macro. Perhaps it's time for me to update this to a function like it is in rocSOLVER.

Reply:
Oh, scratch that, I guess it's already the same as in rocSOLVER. I must be thinking of a different macro that was turned into a function.

What do you think? Should we address the unusedVariable warning or ignore it?

Reply:
The way that cppcheck is being invoked is not ideal. We're not giving it the proper include directories to find all the macro definitions. When we integrate it into our CI builds, it will be done better, and hopefully that issue will go away.

Reply:
As a general rule: don't make your code worse just to satisfy a linter. They exist to bring your attention to something that *might* be a problem. If you've taken a careful look at something cppcheck is complaining about, and it's not giving you good advice, then ignore it.

Reply:
It was only _slightly_ worse, but I agree that this is better. I just need to remember to keep the comments up-to-date.